### ISSUE TITLE
New Certificate Authority SRG Root X1

### ISSUE THREAD

**Issue by karlok05** [ohadschn/letsencrypt-webapp-renewer#117]
https://letsencrypt.org/2020/11/06/own-two-feet.html

**Comment by ohadschn** [comment]
From what I can tell, this means that starting January 2021, this WebJob (as well as the extension it's based on) will keep working, but it will start producing certificates that lead to `ISRG Root X1` (Let's Encrypt own root cert). 

The post you linked to points out the possible compatibility issues that might entail (namely old clients that do not trust that root cert). There is a way to keep getting the old, more compatible certificate, but the latter will expire on Septempber 2021 anyway, so that's only delaying the inevitable. 

In addition, the "alternate" link relation required for the above backward compatible cert between Jan and Sept 2021 does not seem to be supported in neither the extension this WebJob is based on, neither in the actual ACME client (Certes) the latter is using.

So in short, the WebJob will keep working but will produce somewhat less compatible certificates.
